packages/sdk/react-client/src/echo/useSpaces.ts

```diff
@@ -30,12 +31,15 @@ export const useSpace = (spaceKey?: PublicKeyLike) => {
  */
 export const useOrCreateFirstSpace = () => {
```

Comment:
I spoke with @zhenyasav about this. I think we should remove this hook and instead have an option that creates the first space when initially creating the identity.